utils/build.ps1

```diff
@@ -1355,7 +1355,7 @@ function Build-CMakeProject {
       }
       if ($UseBuiltCompilers.Contains("Swift")) {
         Add-KeyValueIfNew $Defines CMAKE_Swift_COMPILER_TARGET (Get-ModuleTriple $Platform)
-        $RuntimeBinaryCache = Get-ProjectBinaryCache $BuildPlatform Runtime
+        $RuntimeBinaryCache = Get-ProjectBinaryCache $Platform Runtime
```

Comment:
Do we have a guarantee that the platform is built before the build platform?

Reply:
That's a fair question, and I suspect the answer is that there's no strong guarantee, but the script currently lists the platforms in the order we need it to (i.e. `$WindowsSDKs` is set to `@("X64", "X86", "Arm64")`), so that's what currently happens since everyone is building on x86-64.

This would be a problem if you were trying to build on an Arm64 box, I guess, in which case `$WindowsSDKs` would need to list `Arm64` first. I don't have an Arm64 Windows box to test on to verify that.

Reply:
Yeah, we definitely build on ARM64 hosts as well. You should be able to build on a parallels VM to validate.

Reply:
> Do we have a guarantee that the platform is built before the build platform?

Looking at the script more carefully, we explicitly build for `$BuildPlatform` first, so the answer to the original question is yes.